src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DefaultProjectSnapshotManagerFactory.cs

```diff
@@ -16,7 +16,8 @@ internal class DefaultProjectSnapshotManagerFactory : ILanguageServiceFactory
         private readonly IEnumerable<ProjectSnapshotChangeTrigger> _triggers;
 
         [ImportingConstructor]
```

Comment:
Some of these guys had bugs in their DI configuration. Thats why there's all the churn here.